index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" />
+    <link rel="stylesheet" href="style.css">
+    <title>BEM FIGMA</title>
+</head>
+
+<body>
+
+    <!--Navbar-->
+    <div class="navbar">
+        <div class="navbar__logo-links">
+            <h3 class="navbar__log">Medical Core</h3>
+            <ul class="navbar__links">
+                <li class="link"><a href="#"></a>Home</li>
```

Comment:
## naming wrong
find the name for the tag `a` because tag without classname is not bem